seed/search.py

```diff
@@ -548,6 +549,9 @@ def build_view_filters_and_sorts(filters: QueryDict, columns: list[dict]) -> tup
     new_filters = Q()
     annotations = {}
     for filter_expression, filter_value in filters.items():
+        if filter_expression[-4:] == '__ne' and filter_value == '':
+            filter_expression = filter_expression.replace('__ne', '__isnull')
+            filter_value = False
```

Comment:
I don't believe this is the most elegant solution, but it should work until that solution is found

Reply:
Eew, yeah, that's pretty gross.  But based on what we talked about I can't see a better way to do this.